common/src/configuration/global.rs

```diff
@@ -614,6 +618,7 @@ fn convert_node_config(network: Network, cfg: Config) -> Result<GlobalConfig, Co
         wallet_base_node_service_request_max_age,
         prevent_fee_gt_amount,
         proxy_host_address,
+        proxy_submit_to_origin,
```

Comment:
I think this will be more aptly named `proxy_submit_to_monerod` as the user have no context about `origin`. The text can instruct the user to makes this false if doing pool merged mining with `self-select` and `submit-to-origin`.

Reply:
Have added comments to the configuration file to explain it.